libs/common/common.babel

```diff
@@ -17688,6 +17688,29 @@
 											</translation>
 										</translations>
 									</concept_node>
+									<concept_node>
+										<name>manage-kbs</name>
+										<description/>
+										<comment/>
+										<translations>
+											<translation>
+												<language>ca-ES</language>
+												<approved>false</approved>
+											</translation>
+											<translation>
+												<language>en-US</language>
+												<approved>false</approved>
+											</translation>
+											<translation>
+												<language>es-ES</language>
+												<approved>false</approved>
+											</translation>
+											<translation>
+												<language>fr-FR</language>
+												<approved>false</approved>
+											</translation>
+										</translations>
+									</concept_node>
```

Comment:
german and portuguese are missing, this will cause the same problem I had earlier, it would be better for you to rebase and add the missing translations